prometheus/alerts/alert.rules.TEMPLATE

```diff
@@ -210,3 +210,11 @@ groups:
         service: cosmos-monitoring
       annotations:
         description: 'Your validator `{{ $labels.moniker }}` is jailed! `{{ $value }}`!'
+
+    - alert: LoadTestClientTxsNotCommitted
+      expr: sum(sum_over_time(loadtest_client_sei_tx_failed[5m]) > 5) by (cluster_name, reason)
+      labels:
+        severity: warning
+        service: loadtest-client
+      annotations:
+        description: 'More than 5 TXs failed in the last 5 minutes in `{{ $labels.cluster_name }}` have failed due to `{{ $labels.reason }}` use `loadtest_client_sei_tx_code` to get more information'
```

Comment:
The TX code seems to be pretty noisy, we get transient insufficient funds and other errors that is inconsistent 
<img width="1737" alt="image" src="https://user-images.githubusercontent.com/18161326/201008617-4c9993b7-099b-46cc-b513-6b814f9979e8.png">